x-pack/plugins/maps/public/_mapbox_hacks.scss

```diff
@@ -10,17 +10,62 @@
 
   .mapboxgl-ctrl-group:not(:empty) {
     @include euiBottomShadowLarge;
+
+    @include kbnThemeStyle($theme: 'v7') {
+      border-radius: $euiBorderRadius;
+    }
+
+    @include kbnThemeStyle($theme: 'v8') {
+      border-radius: $euiBorderRadius * (2 / 3);
+    }
+
     background-color: $euiColorEmptyShade;
-    border-radius: $euiBorderRadius;
+    transition: transform $euiAnimSpeedNormal ease-in-out;
+
+    &:hover {
+      transform: translateY(-1px);
+    }
 
     > button {
       @include size($euiSizeXL);
 
+      @include kbnThemeStyle($theme: 'v7') {
+        &:first-child {
+          border-radius: $euiBorderRadius $euiBorderRadius 0 0;
+        }
+
+        &:last-child {
+          border-radius: 0 0 $euiBorderRadius $euiBorderRadius;
+        }
+      }
+
+      @include kbnThemeStyle($theme: 'v8') {
+        &:first-child {
+          border-radius: ($euiBorderRadius * (2 / 3)) ($euiBorderRadius * (2 / 3)) 0 0;
+        }
+
+        &:last-child {
+          border-radius: 0 0 ($euiBorderRadius * (2 / 3)) ($euiBorderRadius * (2 / 3));
+        }
+      }
```

Comment:
Rather than styling the `border-radius` of these child buttons, could you instead remove these styles and just place an `overflow: hidden;` on the parent `.mapboxgl-ctrl-group` element? I believe that would achieve the same appearance with less SCSS. Thoughts?

Reply:
Thanks @MichaelMarcialis! Great suggestion. I already added. 👍🏽